workers/unity/Packages/com.improbable.gdk.gameobjectcreation/GameObjectCreationSystemHelper.cs

```diff
@@ -8,7 +8,7 @@ public static class GameObjectCreationSystemHelper
     {
         public static void EnableStandardGameObjectCreation(World world, GameObject workerGameObject = null)
         {
-            var workerSystem = world.GetOrCreateManager<WorkerSystem>();
+            var workerSystem = world.GetExistingManager<WorkerSystem>();
```

Comment:
should we throw an exception if this returns null?

Reply:
this should already do that :)

Reply:
`GetExistingManager` won't throw an exception, it will return null